debian/mini-iso-tools.postinst

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+update-initramfs -u
```

Comment:
So I know this is what casper does (because I went and looked <wink>) but there is also a dpkg trigger for rebuilding the initramfs. And if you say "apt install casper mini-iso-tools" it would be nice if we only rebuild the initramfs once. Could you take a few minutes to look into this? (and we should change casper to use this too of course)

Reply:
At recentish debhelper-compat, it handles this.
Fixing casper means moving debhelper-compat which means fixing systemd handling.
The mini-iso-tools part is implemented.